reference/sites.v3.yml

```diff
@@ -732,6 +789,13 @@ components:
       description: The keys and values in an errors object will vary depending on the error received.
       title: _errors
       x-internal: false
+    EmptyResponse:
+      type: object
+      properties:
+        data:
+          type: object
+        meta:
+          type: object
```

Comment:
I see this repeating a lot through -- do we want to ref it rather than reproducing?